src/glue/jobs/s3_to_json.py

```diff
@@ -544,8 +563,21 @@ def main() -> None:
                 Key = message["source_key"]
         )
         s3_obj["Body"] = s3_obj["Body"].read()
+        cohort = None
+        if "adults_v1" in message["source_key"]:
+            cohort = "adults_v1"
+        elif "pediatric_v1" in message["source_key"]:
+            cohort = "pediatric_v1"
+        else:
+            logger.warning(
```

Comment:
Does this warning appear for you in the logs? When i was troubleshooting something with s3 to json, I was reading that because this isn't a spark job, the logging info doesn't appear in cloudtrail in any of the logs or error outputs.

I had to do something like:
```
logger = logging.getLogger()
logger.setLevel(logging.DEBUG)
handler = logging.StreamHandler(sys.stdout)
handler.setLevel(logging.DEBUG)
formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
handler.setFormatter(formatter)
logger.addHandler(handler)
```
To get the logs to show at least at the INFO and DEBUG levels. But maybe WARN level logs are outputted.

Since this seems like a pretty important warning, should we have a test to make sure this is always caught or escalate it to an exception?

Reply:
Nit: Just trying to think ahead in the future, but we might want to make all logs we put into cloudtrail machine parsable.  We can then use mine the logs for issues on a regular basis

Reply:
I was hoping to be productive today but I just spent the past 2 hours trying to figure out why this works 😓 

This logger flow helped my understanding: https://docs.python.org/3/howto/logging.html#logging-flow

So it seems like this would also work:
```
logger = logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
logger.handlers[0].setLevel(logging.DEBUG)
```
but it doesn't.

I also couldn't figure out why a `print` statement I had accidentally left in the `transform_object_to_array_of_objects` function would write to `/aws-glue/python-jobs/output` but a print statement I put at the global namespace wouldn't.

I ended up copying this code verbatim except for the format, which I replaced with the default format. Now I get my `debug` level statements (which I wasn't getting before), but I also get debug statements from every library used in the script, so now there is an overwhelming amount of logging.

Basically, there is no documentation about logging in an Glue python shell job and it is super confusing.